css/styles.css

```diff
@@ -0,0 +1,159 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto&display=swap");
+@import url("https://fonts.googleapis.com/css2?family=Crete+Round:ital@1&display=swap");
+@import url("https://fonts.googleapis.com/css2?family=Poppins:wght@200&display=swap");
+
+body {
+  position: relative;
+  width: 375px;
+  height: 6968px;
+  margin: auto;
+}
+
+.wrapper {
+  background-image: url(../img/backgroung.svg);
+  background-repeat: no-repeat;
+  max-width: 375px;
+}

```

Comment:
- [ ] When adding margins, width, and height to your pages, always favor percent-based units (%, vw, vh) over fixed units (px), except when a fixed unit is really needed. this will help improve the responsiveness of your page.